src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+import debounce from 'lodash/debounce';
+
+import {blockNew} from '../utils/cn';
+
+import {BaseSlider} from './BaseSlider/BaseSlider';
+import {SliderTooltip} from './SliderTooltip/SliderTooltip';
+import type {BaseSliderRefType, RcSliderValueType, SliderProps, SliderValue} from './sliderTypes';
+import {getInnerState} from './utils';
+
+import './Slider.scss';
+
+const b = blockNew('slider');
+
+export const Slider = React.forwardRef(function Slider(
+    {
+        value,
+        defaultValue = 0,
+        size = 'm',
+        min = 0,
+        max = 100,
+        step = 1,
+        infoPointCount = 0,
+        availableValues,
+        withTooltip = false,
+        error,
```

Comment:
`error` -> `errorMessage` + `validationState: 'invalid'`, check the TextInput or TextArea component for reference

Reply:
https://github.com/gravity-ui/uikit/pull/1243/commits/06e0ecbc1b07914adb2114e74cfca1edf80d6f2c